src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -300,11 +300,6 @@ medAbstractWorkspace* medToolBox::getWorkspace()
 void medToolBox::setWorkspace(medAbstractWorkspace* workspace)
 {
     d->workspace = workspace;
-    if (workspace)
-    {
-        connect(workspace->stackedViewContainers(), SIGNAL(containersSelectedChanged()), this,
-                SLOT(updateView()), Qt::UniqueConnection);
-    }
```

Comment:
I've solved the medRoiManagement error in https://github.com/medInria/medInria-private/pull/45/commits/47b00d96c81efb23f7b1596b397f881439441c77